src/javascript/app/pages/user/account/settings/personal_details.js

```diff
@@ -129,6 +129,18 @@ const PersonalDetails = (() => {
         }
     };
 
+    const selectContactCSMessage = (immutable_fields) => {
+        if (/name/.test(immutable_fields.toString()) && /tax/.test(immutable_fields.toString())) {
```

Comment:
thanks for the change
nit suggestion: we can make these regex text into variables e.g `const has_name = /name/.test(immutable_fields.toString())`

Reply:
This is so simple but so important. Don't know how I missed it in the first place. Thank you for pointing at it! All set :)